### ISSUE TITLE
Missing pem files to run largetransfer.lua test with LuaSec

### ISSUE THREAD

**Issue by hishamhm** [keplerproject/copas#36]
Could these test pem files be added to the repository so that the test runs to completion? Thanks!

I'm currently getting this:

```
starting loop
Writing... 49501... Done!	0.11228013038635	nil	10485760
Reading... 49501... Done!	0.15213513374329	nil	10485760
Writing... 49500... Done!	0.15223217010498	nil	10485760
Reading... 49500... Done!	0.19177603721619	closed	10485760
1	minutes:	60.054605960846
Loop done
starting loop
/Users/hisham/.luarocks/share/lua/5.1/copas.lua:369: error loading private key (No such file or directory)	thread: 0x9690af8	tcp{client}: 0x9851fec
/Users/hisham/.luarocks/share/lua/5.1/copas.lua:369: error loading private key (system lib)	thread: 0x96877c8	tcp{client}: 0x984ff84
/Users/hisham/.luarocks/share/lua/5.1/copas.lua:369: error loading private key (system lib)	thread: 0x986d290	tcp{client}: 0x986b23c
/Users/hisham/.luarocks/share/lua/5.1/copas.lua:369: error loading private key (No such file or directory)	thread: 0x96812a0	tcp{client}: 0x9868c64
1	minutes:	60.061335086823
2	minutes:	120.12024402618
3	minutes:	180.17772102356
^C^C
```

**Comment by Tieske** [comment]
The intention was to use the sample certificates by LuaSec. Though I cannot seem to find it documented anywhere.

see https://github.com/brunoos/luasec/tree/master/samples/certs 

does it work if you generate those?

**Comment by hishamhm** [comment]
I generated them all running the *.sh scripts, them copied them to the copas/tests directory. 

Now I got this:

```
/Users/hisham/.luarocks/share/lua/5.1/copas.lua:369: error loading certificate (no start line)	thread: 0x9f26838	tcp{client}: 0xb22d4cc
/Users/hisham/.luarocks/share/lua/5.1/copas.lua:381: closed	thread: 0x9f36b18	SSL connection: 0xb249e34
/Users/hisham/.luarocks/share/lua/5.1/copas.lua:369: error loading certificate (no start line)	thread: 0x9f34180	tcp{client}: 0xb25222c
/Users/hisham/.luarocks/share/lua/5.1/copas.lua:381: closed	thread: 0xb251c98	SSL connection: 0xb2625ac
```

**Comment by Tieske** [comment]
just included the certificates with #37

**Comment by Tieske** [comment]
Can you test again? On a Debian 8 VM it all works for me using the certificates that are now included.

**Comment by Tieske** [comment]
@hishamhm any update on this?

**Comment by Tieske** [comment]
ping

**Comment by mpeterv** [comment]
I can confirm that the tests passes now:

```
starting loop
Writing... 49501... Done!	0.060412168502808	nil	10485760
Reading... 49501... Done!	0.079982042312622	nil	10485760
Writing... 49500... Done!	0.080131053924561	nil	10485760
Reading... 49500... Done!	0.098365068435669	closed	10485760
1	minutes:	60.036839008331
Loop done
starting loop
Writing... 49500... Done!	0.36047005653381	nil	10485760
Reading... 49500... Done!	0.41323804855347	nil	10485760
Writing... 49501... Done!	0.4397439956665	nil	10485760
Reading... 49501... Done!	0.48988103866577	nil	10485760
1	minutes:	60.099549055099
Loop done
```

By the way, is it necessary to wait for a whole minute per loops there? I've changed `copas.sleep(60)` to `copas.sleep(0.5)` and it still works and takes only a second.

**Comment by hishamhm** [comment]
Confirmed, also changing 60 to 0.5!
